lib/serializers/bib.js

```diff
@@ -28,9 +28,6 @@ var fromMarcJson = (object, datasource) => {
     // Bnumber identifier
     builder.add(fieldMapper.predicateFor('Identifier'), { id: object.id, type: 'nypl:Bnumber' }, 0, { path: 'id' })
 
-    // Title
-    if (object.title) builder.add(fieldMapper.predicateFor('Title'), { literal: object.title }, 0, { path: 'title' })
-
```

Comment:
Why is this being removed?

Reply:
Whenever possible we should pull data from varFields instead of Sierra convenience fields like `.title`. Extracting the title via `.title` is equivalent to extracting it via `.varFields(245, ['a','b'])`, which, with [this PR](https://github.com/NYPL/nypl-core/pull/92) is now [the configured mapping path for most institutions](https://github.com/NYPL/nypl-core/blob/3eb03d206a735bc8c0830c8726e50d6f70793a3a/mappings/recap-discovery/field-mapping-bib.json#L1070-L1081) ([Harvard adds a 'c' subfield](https://github.com/NYPL/nypl-core/blob/3eb03d206a735bc8c0830c8726e50d6f70793a3a/mappings/recap-discovery/field-mapping-bib.json#L1082-L1090)). So this is being removed because we're [now extracting it later on](https://github.com/NYPL-discovery/discovery-store-poster/blob/d645f228bf314eb609bf584c2158a64f0d15a7e8/lib/serializers/bib.js#L194) using the general mechanism for extracting straightforward literals.

The reason for making this change now - in addition to being more explicit and protected against vendor/config contingent fields like `.title` - is the system this PR proposes for extracting parallels depends on the marc `path`s in the primary field all be replicable for the parallel fields. If our "Title" mapping specifies querying marc `245 $a $b`, we should be able to find find a `880` varfield (with appropriate `$6` link) and pull subfields `$a $b`. If the "Title" mapping uses the `.title` property, there's no formal link to the parallel field.